src/mongocxx/options/find_one_and_update.hpp

```diff
@@ -27,6 +26,17 @@ MONGOCXX_INLINE_NAMESPACE_BEGIN
 namespace options {
 
 ///
+/// Argument for find_one_and_update and find_one_and_replace to determine
+/// whether to return the document before or after modification.
+///
+/// @see https://docs.mongodb.org/manual/reference/method/db.collection.findAndModify/#return-data
+///
+enum class return_document {
```

Comment:
I misunderstood, I didn't realize that find_one_and_modify had no matching operation, or .cpp file.  It is actually a common header for the other find_one_and* operation options.  I think we should do as you said and just remove the comment.  I also think we should rename this file to something clearer, maybe `find_one_common_options.hpp`